views/404.ejs

```diff
@@ -0,0 +1,54 @@
+<html>
+  <head>
+    <meta charset='utf-8'>
+    <title><%-data.name%></title>
+    <style>
+    * {
```

Comment:
Please share the styles between templates. This is typically done by reading the style from a shared file, and then sending it in `data` to the template.